nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>This is actually a kind of alias for #| notest  it does the same thing.  If you don't mind can you please verify this.  With this in mind you might want to combine the explanation of #| notest and #| eval: &lt;true|false&gt; </p><p> </p><p>nbdev respects both.  The latter is a quarto directive.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>